src/features/main/right-section/right-section.tsx

```diff
@@ -0,0 +1,123 @@
+import React, { useState } from 'react';
+import Tabs from '@mui/material/Tabs';
+import Tab from '@mui/material/Tab';
+import Box from '@mui/material/Box';
+import {useTheme} from "@mui/material";
+
+// const useStyles = makeStyles((theme: Theme) => ({
+//     root: {
+//         display: 'flex',
+//         height: '100vh',
+//         width: '300px',
+//         backgroundColor: theme.palette.grey[200],
+//     },
+//     tabs: {
+//         borderRight: `1px solid ${theme.palette.divider}`,
+//     },
+//     tabContent: {
+//         flexGrow: 1,
+//         padding: theme.spacing(3),
+//     },
+// }));
+
+export const RightSection = () => {
+    const theme = useTheme();
+    const [value, setValue] = useState(0);
+
+    const handleChange = (event: React.SyntheticEvent, newValue: number) => {
+        setValue(newValue);
+    };
+
+    return (
+        <Box sx={{
+            display: 'flex',
+            height: '100vh',
+            width: '4rem',
+            backgroundColor: theme.palette.mode === 'dark' ? "black" : "#f5f4f4f7",
```

Comment:
You can use theme.palette.common.black instead of "black". Could we also not use random colors like "#f5f4f4f7". Add to the theme or at least save it as a const with a name that will describe what this color is responsible for